### PR TITLE
Remove hwapi package

### PR DESCRIPTION
**Pull request by ChriMarMe**
Add go-linux-lowlevel-hw as dependency
Integrate new dependency
Pray for salvation

Signed-off-by: Christopher Meis <christopher.meis@9elements.com>

### CODE REVIEW

**Comment by ChriMarMe** [comment]
This PR should get new release, because it has breaking changes.

File: pkg/registers/marshalling_test.go [hunk 1]
```diff
@@ -1,10 +1,11 @@
 package registers_test
 
 import (
-	"github.com/9elements/converged-security-suite/v2/pkg/hwapi"
-	"github.com/9elements/converged-security-suite/v2/pkg/registers"
 	"reflect"
 	"testing"
+
+	hwInternal "github.com/9elements/converged-security-suite/v2/pkg/hwapi"
```
Comment (rihter007) [inline pkg/registers/marshalling_test.go:7]: I guess hwInternal is not in go-style and hwapi looks a bit better (I might be wrong)

File: pkg/test/acpi.go [hunk 2]
```diff
@@ -453,35 +453,36 @@ func CheckRSDTValid(txtAPI hwapi.APIInterfaces, config *tools.Configuration) (bo
 }
 
 //CheckXSDTValid tests if the XSDT ACPI table is vaid
```
Comment (rihter007) [inline pkg/test/acpi.go:455]: typo "vaid" -> "valid"

File: pkg/test/acpi.go [hunk 3]
```diff
@@ -453,35 +453,36 @@ func CheckRSDTValid(txtAPI hwapi.APIInterfaces, config *tools.Configuration) (bo
...
+	if os.IsNotExist(err1) && os.IsNotExist(err2) {
 		return false, fmt.Errorf("ACPI table XSDT is invalid"), nil
-	} else if err != nil {
-		return false, nil, err
+	} else if err1 != nil && err2 != nil {
+		return false, nil, fmt.Errorf("%v and %v", err1, err2)
 	}
 
-	return true, nil, nil
+	return false, nil, nil
```
Comment (rihter007) [inline pkg/test/acpi.go:465]: when do we return true? :)

**Review by rihter007 — CHANGES_REQUESTED** [review]
Minor code-style + CheckXSDTValid now always returns false.

File: pkg/registers/marshalling_test.go (hunk 1 above)
Comment (ChriMarMe) [reply pkg/registers/marshalling_test.go]: addressed

File: pkg/test/acpi.go (hunk 2 above)
Comment (ChriMarMe) [reply pkg/test/acpi.go]: addressed

File: pkg/test/acpi.go (hunk 3 above)
Comment (ChriMarMe) [reply pkg/test/acpi.go]: addressed

**Review by xaionaro — APPROVED** [review]

**Review by rihter007 — APPROVED** [review]
LGTM, thank you!

**Review by xaionaro — APPROVED** [review]

**Review by ArthurHeymans — APPROVED** [review]

**Pull request merged**